src/ontology/mondo-edit.obo

```diff
@@ -329321,26 +329321,27 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 [Term]
 id: MONDO:0014937
 name: aniridia 2
+comment: The new term, ocular dysgenesis caused by defects in PAX6 regulation, was proposed by the ClinGen Glaucoma / Neuro-Ophthalmology GCEP as a parent term for cases of ocular disease caused by variants in the PAX6 gene itself (MONDO:0800183 PAX6-related ocular dysgenesis) or by disruption of PAX6 expression by variants in the ELP4 locus (this class, MONDO:0014937 aniridia 2) or TRIM44 locus (MONDO:0014938 aniridia 3).
 subset: gard_rare {source="GARD:16200"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "AN2" EXACT ABBREVIATION [OMIM:617141]
 synonym: "aniridia 2" EXACT [OMIM:617141]
-synonym: "aniridia 2; AN2" EXACT []
 synonym: "aniridia type 2" EXACT [MONDORULE:1, OMIM:617141]
 xref: GARD:16200 {source="OMIM:617141"}
 xref: MESH:C536372 {source="MONDO:equivalentTo"}
 xref: OMIM:617141 {source="MONDO:equivalentTo"}
 xref: SCTID:253232000 {source="MONDO:equivalentTo"}
 xref: UMLS:C0344543 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 is_a: MONDO:0007119 {source="OMIM:617141", source="Orphanet:250923/btnt"} ! isolated aniridia
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1171 {source="MONDO:mim2gene_medgen"} ! ELP4
```

Comment:
I don't think it is correct to remove this axiom. ELP4 is reported in OMIM:617141 to be associated with this disease.